homework/hw01/task5.py

```diff
@@ -0,0 +1,30 @@
+# 5. Программа загадывает число от 0 до 1000. Необходимо угадать число за 10 попыток.
+# Программа должна подсказывать «больше» или «меньше» после каждой попытки.
+# Для генерации случайного числа используйте код:
+#
+# ```
+# from random import randint
+# num = randint(LOWER_LIMIT, UPPER_LIMIT)
+# ```
+
+from random import randint
+
+LOWER_LIMIT = 0
+UPPER_LIMIT = 1000
+STEP_MAX = 10
+
+num = randint(LOWER_LIMIT, UPPER_LIMIT)
+# print(f'ЗАГАДАННОЕ ЧИСЛО: {num}')
+
+while STEP_MAX > 0:
+    step = int(input(f'\nОставшееся количество ходов {STEP_MAX}, введите число: '))
+    STEP_MAX -= 1
+    if STEP_MAX == 0:
+        print(f'Игра окончена. Загаданное число было {num}.')
+    elif step == num:
+        print('\nВы угадали!\n')
+        break
+    elif step > num:
+        print('> Загаданное число меньше.')
+    else:
+        print('> Загаданное число больше.')
```

Comment:
На мой взгляд, лучше использовать связку `break` и `else` от цикла
```suggestion
while STEP_MAX:
    step = int(input(f'\nОставшееся количество ходов {STEP_MAX}, введите число: '))
    STEP_MAX -= 1
    if step == num:
        print('\nВы угадали!\n')
        break
    elif step > num:
        print('> Загаданное число меньше.')
    else:
        print('> Загаданное число больше.')
else:
    print(f'Игра окончена. Загаданное число было {num}.')
```